src/components/v2/V2Project/V2ProjectHeaderActions.tsx

```diff
@@ -61,6 +75,15 @@ export default function V2ProjectHeaderActions() {
           />
         </Tooltip>
         {showReconfigureButton && <V2ReconfigureFundingModalTrigger />}
+        {settingsPageEnabled && (
+          <Tooltip title={t`Open settings page`} placement="bottom">
+            <Button
+              icon={<InfoCircleOutlined />}
+              type="text"
+              onClick={handleSettingsPageButtonClick}
```

Comment:
**suggestion** for accessibility, lets make this a real link. Not exactly sure if Next Link will work, but `<Button href="/v2/p/${projectId}/settings">` does.